pip_check_reqs/common.py

```diff
@@ -77,41 +77,32 @@ def _add_module(self, modname: str, lineno: int) -> None:
             return
         path = None
         progress = []
-        modpath = last_modpath = None
+        modpath = None
         for modname_part in modname.split("."):
-            try:
-                _, modpath, _ = imp.find_module(modname_part, path)
-            except ImportError:
-                # the component specified at this point is not importable
-                # (is just an attribute of the module)
-                # *or* it's not actually installed, so we don't care either
-                break
-
-            # success! we found *something*
-            progress.append(modname_part)
+            find_spec_result = importlib.util.find_spec(
```

Comment:
Nitpick: `find_spec` result is called ["module spec"](https://docs.python.org/3/glossary.html#term-module-spec).

Reply:
I'll fix that!